docs/zephyr/tutorial/pins.rst

```diff
@@ -0,0 +1,33 @@
+.. _pins_zephyr:
+
+GPIO Pins
+=========
+
+Use :ref:`machine.Pin <machine.Pin>` to control I/O pins.
+
+For Zephyr, pins are initialized using a tuple of port and pin number ``(\"GPIO_x\", pin#)``
+for the ``id`` value. For example to initialize a pin for the red LED on a FRDM-k64 board::
+
+        LED = Pin(("GPIO_1", 22), Pin.OUT)
+        
+Reference your board's datasheet or Zephyr documentation for pin numbers.
```

Comment:
The pin labelling is hard to get right.  I think it'd be good to provide more info here, eg for a selection of boards what the format/style is for the pin names/numbers.